packages/react-app/src/components/pages/Bounties/index.tsx

```diff
@@ -48,10 +53,17 @@ const Bounties = ({ id }: PreFilterProps): JSX.Element => {
 		window.scrollTo(0, 0);
 	};
 
+	let dynamicUrl = '/api/bounties';
```

Comment:
Broke up the dynamic URL as it was long af, added customer ID as a query param when fetching bounties